examples/demo.rs

```diff
@@ -502,18 +530,39 @@ fn on_touch_handler(app: &mut appctx::ApplicationContext, input: multitouch::Mul
             if !CANVAS_REGION.contains_point(&position.cast().unwrap()) {
                 return;
             }
-            let position_float = position.cast().unwrap();
             let rect = match G_TOUCH_MODE.load(Ordering::Relaxed) {
-                TouchMode::Bezier => framebuffer.draw_bezier(
-                    position_float,
-                    position_float + cgmath::vec2(155.0, 14.0),
-                    position_float + cgmath::vec2(200.0, 200.0),
-                    2.0,
-                    color::BLACK,
-                ),
+                TouchMode::Bezier => {
+                    let position_float = position.cast().unwrap();
+                    framebuffer.draw_bezier(
+                        position_float,
+                        position_float + cgmath::vec2(155.0, 14.0),
```

Comment:
These values were quite arbitrary. Feel free to draw a more interesting curve here.

Reply:
Yeah, a lot more interesting for sure.

[As seen here](https://i.imgur.com/AK2cU3O.jpg)

Reply:
That looks right to me! Interesting was my goal. I actually drew a scribble and then modelled the curve on that, to get a fairly natural flow.

Now, is that the kind of interesting you were looking for?